lib/python/picongpu/plugins/plot_mpl/phase_space_visualizer.py

```diff
@@ -54,13 +54,20 @@ def _create_plt_obj(self, ax):
             origin='lower',
             norm=LogNorm()
         )
+        self.cbar = plt.colorbar(self.plt_obj, ax=ax)
+        self.cbar.set_label(
+                r'$Q / \mathrm{d}r \mathrm{d}p$ [$\mathrm{C s kg^{-1} m^{-2}}$] ')
+        ax.set_xlabel(r'${0}$ [${1}$]'.format(meta.r, "\mathrm{\mu m}"))
+        ax.set_ylabel(r'$p_{0}$ [$\beta\gamma$]'.format(meta.p))
 
     def _update_plt_obj(self):
         """
         Implementation of base class function.
         """
         dat, meta = self.data
         self.plt_obj.set_data(np.abs(dat).T * meta.dV)
+        self.plt_obj.autoscale()
+        self.cbar.update_normal(self.plt_obj)
```

Comment:
question:
does this line solve you rescaling issue?

Reply:
no, the line before updates the ranges and that one updates the cb drawing

Reply:
thanks